ogre2/src/Ogre2GpuRays.cc

```diff
@@ -661,7 +661,22 @@ void Ogre2GpuRays::Destroy()
     ogreCompMgr->removeWorkspace(this->dataPtr->ogreCompositorWorkspace2nd);
     this->dataPtr->ogreCompositorWorkspace2nd = nullptr;
   }
+
+  if (this->scene)
```

Comment:
Should we also check this?

```suggestion
  if (this->scene && this->dataPtr->ogreCamera)
```

Reply:
there is a [check](https://github.com/gazebosim/gz-rendering/blob/f2f2ced9fe322f9897f752b731e7508665651025/ogre2/src/Ogre2GpuRays.cc#L588) earlier in the function that should cover this